index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
kindly format your document

Reply:
- [ ] Please be more specific and explain what you want simply and Kindly mention the reason why the reviewee had to do the change.

Reply:
- [ ] Kindly implement the changes asked by the previous reviewer and mention why you want the changes. Instead of just writing `kindly format your document`, make sure to mention why you are suggesting this. Like you can write `Kindly format your document following the HTML best practices.`

Reply:
hi @sja-thedude I have done the requested changes.